### PR TITLE
Add jenkins

### PR DESCRIPTION
**Pull request by tomiir**
## Summary
Added jenkins and config files

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@Library('wolox-ci') _
+
+node {
+
+  checkout scm
+
+  woloxCi('.woloxci/config.yml');
+}
```
Comment (facucompe) [inline Jenkinsfile]: Perdón, es un TOC
```suggestion
}

```

File: .woloxci/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM node:10.11.0-alpine
```
Comment (damfinkel) [inline .woloxci/Dockerfile]: I'd use 12.14.1-alpine, which is the latest stable release: https://nodejs.org/en/

**Review by damfinkel — CHANGES_REQUESTED** [review]

**Review by damfinkel — APPROVED** [review]

**Review by pabloferro — APPROVED** [review]

**Review by carlosbollero — APPROVED** [review]
A comment for further developments.
What about adding a step to create a new project, build and run it, then check there is no errors?
This way we will be sure any change we do, doesn't break future bootstrapped projects.
What do you think? @tomiir @pabloferro @damfinkel

**Comment by tomiir** [comment]
@pabloferro  @damfinkel  I'm merging this but I think what @carlosbollero  makes a lot of sense. Will add a card to the trello board so we don't forget about this.

**Pull request merged**